src/rules/it.toml

```diff
@@ -4,16 +4,11 @@ max_word_count = 16
 min_characters = 2
 may_end_with_colon = false
 quote_start_with_letter = true
-needs_punctuation_end = false
+needs_punctuation_end = true
 needs_letter_start = true
 needs_uppercase_start = true
-disallowed_symbols = [
-  '<', '>', '+', '*', '\', '#', '@', '^', '[', ']', '(', ')', '/', '»', '«',
-  'â', 'ă', 'ș', 'ç', 'Å', 'ý', 'ö', 'ü', 'ā', 'ö', '%', '&',
-  'α', 'β', 'Γ', 'γ', 'Δ', 'δ', 'ε', 'ζ', 'η', 'Θ', 'θ', 'ι', 'κ',
-  'Λ', 'λ', 'μ', 'ν', 'Ξ', 'ξ', 'Π', 'π', 'ρ', 'Σ', 'σ', 'ς', 'τ',
-  'υ', 'Φ', 'φ', 'χ', 'Ψ', 'ψ', 'Ω', 'ω', '©'
-]
+allowed_symbols_regex="[A-Za-zìàèòùáéíóúÁÉÍÓÚÀÌÒÈ;‚'–\\. \"\\?!]"
+disallowed_symbols = []
 broken_whitespace = ["  ", " ,", " .", " ?", " !", " ;", " :"]
-disallowed_words = []
+disallowed_words = ['Jr', 'vs', 'vol', 'Av', 'Blvd', 'aprox', 'Cd', 'Cen', 'Ant', 'etc', 'art', 'Ltd', 'Co', 'esq', 'Corp', 'Dra', 'Prof', 'Urb', 'Bs', 'Mr', 'Mrs', 'subsp', 'p.e','s.s', 'Av', 'Blvd', 'Arq', 'St','Sta', 'S.', 'Sr.', 'Dr.' ]
```

Comment:
Why "Prof" is in the disallowed?
It's an "abbreviation" (actually it's an apocope -> troncamento)

It's like "signore"->"signor", etc. both forms are correct

The "Sta" abbreviation could mean also the verb "stare" (Io sto, tu stai, egli/ella sta, ...) Present, 3rd singular

Reply:
I would add to "disallowed words" also "I.", "II.", "III.", ..., "XX." (I think until the 20th is enought).
Maybe also "1°", "2°", ..., "20°"